c/openssl/s3_srvr.blast.01.i.cil-1.c

```diff
@@ -1074,6 +1074,25 @@ int main(void)
   s->ctx = malloc(sizeof(SSL_CTX));
   s->session = malloc(sizeof(SSL_SESSION));
   s->state = 8464;
+
+	s->info_callback = (void (*) ()) __VERIFIER_nondet_ulong();
```

Comment:
I was puzzled by this initialization, but `info_callback` is only compared to 0, so this should actually work fine.